src/main/java/dev/mayuna/nostalgiadiscord/discord/DiscordBot.java

```diff
@@ -195,7 +197,7 @@ public static void sendMessageToMinecraftServer(Member member, String message) {
         Logger.info("[DISCORD] " + role + user.getName() + "#" + user.getDiscriminator() + " > " + message);
 
         for (Player player : Bukkit.getOnlinePlayers()) {
-            player.sendMessage("§9[DISCORD] §a" + role + "§e" + user.getName() + "§8#" + user.getDiscriminator() + " §7» §f" + message);
+            player.sendMessage(ChatColor.translateAlternateColorCodes('&',"&9[DISCORD] &a" + role + "&e" + user.getName() + "&8#" + user.getDiscriminator() + " &7» &f" + message));
```

Comment:
I don't really see any reason behind this. This method just replaces & with § really

```java
public static String translateAlternateColorCodes(char altColorChar, String textToTranslate) {
    char[] b = textToTranslate.toCharArray();

    for(int i = 0; i < b.length - 1; ++i) {
        if (b[i] == altColorChar && "0123456789AaBbCcDdEeFfKkLlMmNnOoRr".indexOf(b[i + 1]) > -1) {
            b[i] = 167;
            b[i + 1] = Character.toLowerCase(b[i + 1]);
        }
    }

    return new String(b);
}
```